react/src/pages/NotificationPopover.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles, Paper, Box, Typography, Tooltip, IconButton, Popover } from '@material-ui/core';
+import { NotificationsActive } from '@material-ui/icons';
+import Notification from './Notification';
+import { AnalysisRow } from './analysis/Analysis';
+
+const useStyles = makeStyles(theme => ({
+    popover: {
+        minWidth: '450px',
+    },
+    paper: {
+        padding: theme.spacing(2), 
+    },
+    notifications: {
+        '& > * + *': {
+          marginTop: theme.spacing(2),
+        },
+        maxHeight: '500px',
+    },
+    title: {
+        flexGrow: 1,
+        paddingLeft: theme.spacing(1),
+    },
+    titleBox: {
+        display: 'flex',
+        alignItems: 'center',
+        paddingBottom: theme.spacing(1),
+    },
+}));
+
+export interface NotificationPopoverProps {
+    lastLoginTime: string;
+}
+
+export default function NotificationPopover({ lastLoginTime }: NotificationPopoverProps) {
+    const classes = useStyles();
+    const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+    const [analyses, setAnalyses] = useState<AnalysisRow[]>([] as AnalysisRow[]);
+    const popoverOpen = Boolean(anchorEl);
+
+    const handlePopoverOpen = (event: React.MouseEvent<HTMLButtonElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handlePopoverClose = () => {
+        setAnchorEl(null);
+    };
+    useEffect(() => {
+        const lastLoginISO = new Date(lastLoginTime).toISOString().slice(0, -1);
+        fetch(`/api/analyses?since=${lastLoginISO}`).then(async response => {
+            if (response.ok) {
+                setAnalyses((await response.json()) as AnalysisRow[]);
+            } else {
+                console.error(`GET /api/analyses?since=ISO_TIMESTAMP failed with ${response.status}: ${response.statusText}`);
+            }
+        });
+    }, []);
```

Comment:
Add `lastLoginTime` to the dependency array. Between this and the other issue, it's causing a freeze and crash when actual data is returned from the endpoint. To test with a login timestamp, you can try something like this in your shell:
```bash
docker exec -it st2020_app_1 python

from datetime import datetime
from app import models, db
user = models.User.query.get(1)
user.last_login = datetime(2020, 1, 1)
db.session.commit()
```